modules/aleph/src/Aleph/AlephClient.php

```diff
@@ -98,7 +99,7 @@ public function request($method, $operation, array $params = array()) {
     }
 
     // Throw exception if the status from Aleph is not OK.
-    throw new \RuntimeException('Request error: ' . $response->code . $response->error);
+    throw new AlephClientException('Request error: ' . $response->code . $response->error);
```

Comment:
[The default Exception constructor already have separate arguments for code and error message](http://php.net/manual/en/language.exceptions.extending.php). You may not want to combine them into a single string unless there is a reason to.

Reply:
You changed this on L101 but not here?